notebooks/223-text-prediction/223-text-prediction.ipynb

```diff
@@ -0,0 +1,773 @@
+{
```

Comment:
<blockquote><a href="https://d4mucfpksywv.cloudfront.net/better-language-models/language_models_are_unsupervised_multitask_learners.pdf" target="_blank">GPT-2</a>, <a href="https://zenodo.org/record/5297715#.ZAmpsXZBztU" target="_blank">GPT-Neo</a>, and <a href="https://arxiv.org/abs/2110.12949v1" target="_blank">PersonaGPT</a>,  </blockquote><p>is it possible to provide all 3 links on arxive papers instead of different sources?</p><p> </p><blockquote>. GPT-2 is available from <a href="https://huggingface.co/gpt2" target="_blank">HuggingFace</a>,  GPT-Neo is also available from <a href="https://huggingface.co/EleutherAI/gpt-neo-125M" target="_blank">HuggingFace</a>.</blockquote><p>I think, more details about models provided in HuggngFace model cards:</p><ul><li> <a href="https://huggingface.co/gpt2" target="_blank">GPT-2</a></li><li><a href="https://huggingface.co/EleutherAI/gpt-neo-125M" target="_blank">GPt-Neo</a></li></ul><p>will be better wording for providing HF models links.</p><p> </p><blockquote>The following image illustrates the complete demo pipeline used for generation: </blockquote><p>but you have 2 images, not clear which one do you mean?</p><p>I think you need to add some explanation that tutorial can work in 2 modes:</p><p>interactive conversation using personaGPT and conditional text generation using GPT2 or GPT-Neo and then provide explanation how it is work with flow chart illustration for each mode</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/897/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I tried to find the arxiv links for the other models but their official links are the ones that i have added, I will look again and try to get the arxiv links